metricbeat/module/prometheus/_meta/config.yml

```diff
@@ -7,3 +7,8 @@
   #namespace: example
   #username: "user"
   #password: "secret"
+
+  # This can be used for service account based authorization:
```

Comment:
Should there also be an update to the reference config files? Also applies to the k8s one.

For the paths here for the prometheus config: Should we have k8s as default ones?

Reply:
I think Prometheus is mostly used in k8s scenarios, it should not harm other use cases anyway? I've pushed a commit to include it in